src/scripts/main.js

```diff
@@ -21,7 +21,26 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  if (typeof data !== 'object' || data === null) {
+    return;
+  }
+
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    if (data.hasOwnProperty(key)) {
+      const li = document.createElement('li');
+
+      li.textContent = key;
+
+      if (typeof data[key] === 'object' && data[key] !== null) {
+        createTree(li, data[key]);
+      }
+
+      element.appendChild(ul);
```

Comment:
The `element.appendChild(ul)` statement is currently inside the loop, which causes the `ul` to be appended multiple times, once for each key in the `data` object. This should be moved outside the loop to ensure the `ul` is appended only once to the `element`.